google_compute_engine/metadata_scripts/script_retriever.py

```diff
@@ -135,7 +135,10 @@ def _DownloadScript(self, url, dest_dir):
     match = gs_regex.match(url)
     if match:
       gs_url = r'gs://%s/%s' % (match.group('bucket'), match.group('obj'))
-      return self._DownloadGsUrl(gs_url, dest_dir)
+      dest = self._DownloadGsUrl(gs_url, dest_dir)
+      # In case gsutil is not installed, continue as a normal URL
+      if dest is not None:
+        return dest
```

Comment:
That would be for every case where we try to use the Google Storage URL function. I think it makes the most sense to do it once in the function return as a fallback.